datadog_checks_base/tests/models/config_models/deprecations.py

```diff
@@ -8,7 +8,6 @@
 #     ddev -x validate models -s <INTEGRATION_NAME>
 
 
-
```

Comment:
Hum, it seems that `ddev validate models datadog_checks_base -s` will add the new line back 🤔